resources/profiles/BIQU.json

```diff
@@ -0,0 +1,194 @@
+            "name": "fdm_process_biqu_common",
+            "sub_path": "process/fdm_process_biqu_common.json"
+        },
+		{
+            "name": "fdm_process_hurakan_common",
+            "sub_path": "process/fdm_process_hurakan_common.json"
+        },
+        {
+            "name": "0.12mm Fine @BX",
+            "sub_path": "process/0.12mm Fine @BIQU BX.json"
+        },
+        {
+            "name": "0.15mm Optimal @BX",
+            "sub_path": "process/0.15mm Optimal @BIQU BX.json"
+        },
+        {
+            "name": "0.20mm Standard @BX",
+            "sub_path": "process/0.20mm Standard @BIQU BX.json"
+        },
+        {
+            "name": "0.24mm Draft @BX",
+            "sub_path": "process/0.24mm Draft @BIQU BX.json"
+        },
+		{
+            "name": "0.12mm Fine @B1",
+            "sub_path": "process/0.12mm Fine @BIQU B1.json"
+        },
+        {
+            "name": "0.15mm Optimal @B1",
+            "sub_path": "process/0.15mm Optimal @BIQU B1.json"
+        },
+        {
+            "name": "0.20mm Standard @B1",
+            "sub_path": "process/0.20mm Standard @BIQU B1.json"
+        },
+        {
+            "name": "0.24mm Draft @B1",
+            "sub_path": "process/0.24mm Draft @BIQU B1.json"
+        },
+		{
+            "name": "0.12mm Fine @Hurakan",
+            "sub_path": "process/0.12mm Fine @BIQU Hurakan.json"
+        },
+        {
+            "name": "0.15mm Optimal @Hurakan",
+            "sub_path": "process/0.15mm Optimal @BIQU Hurakan.json"
+        },
+        {
+            "name": "0.20mm Standard @Hurakan",
+            "sub_path": "process/0.20mm Standard @BIQU Hurakan.json"
+        },
+        {
+            "name": "0.24mm Draft @Hurakan",
+            "sub_path": "process/0.24mm Draft @BIQU Hurakan.json"
+        }
+    ],
+    "filament_list": [
+        {
+            "name": "fdm_filament_common",
+            "sub_path": "filament/fdm_filament_common.json"
+        },
+        {
+            "name": "fdm_filament_abs",
+            "sub_path": "filament/fdm_filament_abs.json"
+        },
+        {
+            "name": "fdm_filament_asa",
+            "sub_path": "filament/fdm_filament_asa.json"
+        },
+        {
+            "name": "fdm_filament_pa",
+            "sub_path": "filament/fdm_filament_pa.json"
+        },
+		{
+            "name": "fdm_filament_pc",
+            "sub_path": "filament/fdm_filament_pc.json"
+        },
+        {
+            "name": "fdm_filament_pet",
+            "sub_path": "filament/fdm_filament_pet.json"
+        },
+		        {
+            "name": "fdm_filament_pla",
+            "sub_path": "filament/fdm_filament_pla.json"
+        },
+		{
+            "name": "fdm_filament_pva",
+            "sub_path": "filament/fdm_filament_pva.json"
+        },
+        {
+            "name": "fdm_filament_tpu",
+            "sub_path": "filament/fdm_filament_tpu.json"
+        },
+        {
+            "name": "BIQU Generic ABS",
```

Comment:
Inconsistent filament name, the name defined inside `BIQU Generic ABS.json` file is `Generic ABS @BIQU`
I don't think it will work

Reply:
Same for all other filament name